packages/react/src/components/CodeSnippet/CodeSnippet-story.js

```diff
@@ -7,51 +7,62 @@
 
 import React from 'react';
 import { action } from '@storybook/addon-actions';
-import { withKnobs, boolean, text } from '@storybook/addon-knobs';
+import { withKnobs, boolean, text, select } from '@storybook/addon-knobs';
 import CodeSnippet from '../CodeSnippet';
 import CodeSnippetSkeleton from './CodeSnippet.Skeleton';
+import mdx from './CodeSnippet.mdx';
 
-const props = {
-  inline: () => ({
-    light: boolean('Light variant (light)', false),
-    feedback: text('Feedback text (feedback)', 'Copied to clipboard'),
-    onClick: action('onClick'),
-    copyLabel: text(
-      'ARIA label for the snippet/copy button (copyLabel)',
-      'copyable code snippet'
-    ),
-    hideCopyButton: boolean('Hide copy button (hideCopyButton)', false),
-  }),
-  single: () => ({
-    light: boolean('Light variant (light)', false),
-    feedback: text('Feedback text (feedback)', 'Copied to clipboard'),
-    copyButtonDescription: text(
-      'Copy icon description (copyButtonDescription)',
-      'copyable code snippet'
-    ),
-    ariaLabel: text(
-      'ARIA label of the container (ariaLabel)',
-      'Container label'
-    ),
-    hideCopyButton: boolean('Hide copy button (hideCopyButton)', false),
-    onClick: action('onClick'),
-  }),
-  multiline: () => ({
-    light: boolean('Light variant (light)', false),
-    feedback: text('Feedback text (feedback)', 'Copied to clipboard'),
-    showMoreText: text(
-      'Text for "show more" button (showMoreText)',
-      'Show more'
-    ),
-    showLessText: text(
-      'Text for "show less" button (showLessText)',
-      'Show less'
-    ),
-    hideCopyButton: boolean('Hide copy button (hideCopyButton)', false),
-    onClick: action('onClick'),
-  }),
+export default {
+  title: 'CodeSnippet',
+  component: CodeSnippet,
+  decorators: [withKnobs],
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+  },
 };
 
+export const codeSnippet = () => (
+  <CodeSnippet type="inline" feedback="Copied to clipboard">
+    {'node -v'}
+  </CodeSnippet>
+);
+
+export const inline = () => (
+  <CodeSnippet type="inline" feedback="Copied to clipboard">
+    {'node -v'}
+  </CodeSnippet>
+);
+
+export const multiline = () => (
+  <CodeSnippet type="multi" feedback="Copied to clipboard">
+    {`@mixin grid-container {
+     width: 100%;
+     padding-right: padding(mobile);
+     padding-left: padding(mobile);`}
+  </CodeSnippet>
+);
+
+export const singleline = () => (
+  <CodeSnippet type="single" feedback="Copied to clipboard">
+    {'node -v'}
+  </CodeSnippet>
+);
+
+export const skeleton = () => (
+  <div style={{ width: '800px' }}>
+    <CodeSnippetSkeleton type="single" style={{ marginBottom: 8 }} />
+    <CodeSnippetSkeleton type="multi" />
+  </div>
+);
+
+export const light = () => (
```

Comment:
Do we need this variation to be displayed? I think we could probably just define this in the Prop documentation like so https://deploy-preview-6606--carbon-components-react.netlify.app/?path=/docs/contentswitcher--default#contentswitcher-light